src/components/DocWrapper/DocList/DocList.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import DocList, {DocListProps} from './index';
+
+describe('DocList', () => {
+    const props: DocListProps = {
+    className: 'test-class',
```

Comment:
The indentation here should be two spaces after `const`

Reply:
also please move the `const props` out of the describe please